service/build.gradle

```diff
@@ -244,14 +244,9 @@ dependencies {
   testImplementation group: 'org.apache.pdfbox', name: 'pdfbox', version: '2.0.19'
 
   testCompile (group: 'org.springframework.boot', name: 'spring-boot-starter-test') {
-    exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
```

Comment:
This was previously excluded due to issues with the mutation testing, might want to check that it still runs if you are removing this part